src/common.h

```diff
@@ -366,6 +366,7 @@ struct module_data {
 	int gvolbase;			/* Global volume base */
 	int gvol;			/* Global volume */
 	const int *vol_table;		/* Volume translation table */
+	int amplify;			/* Per-format amplification adjustment */
```

Comment:
Right now libxmp only uses the IT master volume field to tracker ID SPC conversions. I think you're right and this might be a possible alternative for this feature. I need to make sure the mixer can handle volumes over 1024 first.

Reply:
Notes for implementing this:
* There's no standard for what is considered "100%" volume with IT master volume. Schism and OpenMPT apparently treat 256 as playing at -0dB.
* OpenMPT can use values up to 255 here, which needs a fix in the IT loader: values above 128 are currently rejected.
* Currently, libxmp mixes at -0dB multiplied by channel volume 1024 and right shifted (12 - amplification factor). The default amplification factor is 1, so this roughly works out to dividing each sample by 2. The amplify hack effectively just doubles the output level.
* So, the equivalent would be to set the gain to 2x whatever is considered "100%" (which probably ought to also be configurable, and presumably set to 100 or 128 by default). When `gain` != `gain_default`, apply a final `volume = volume * gain / gain_default;` to the volume calculation.
* libxmp's `XMP_PLAYER_VOLUME` setting is ranged from 0-200 with 100=100%. It would work in a pinch here, but I think having them be separate might be best.
* Implementing this would also fix the underlying issue from #499; this IT uses OpenMPT's default master volume 48.